pvlib/shading.py

```diff
@@ -342,3 +342,144 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
     # Eq. (5); angle between sun's beam and surface
     theta_T = np.degrees(np.arctan2(sx_prime, sz_prime))
     return theta_T
+
+
+def shaded_fraction1d(
+    solar_zenith,
+    solar_azimuth,
+    trackers_axis_azimuth,
+    shaded_tracker_tilt,
+    *,
+    collector_width,
+    row_pitch,
+    surface_to_axis_offset=0,
+    cross_axis_slope=0,
+    shading_tracker_tilt=None,
+):
+    r"""
+    Shaded fraction in the vertical dimension of tilted rows.
+
+    If ``shading_tracker_tilt`` isn't provided, assumes both the shaded
+    row and the one blocking the direct beam
+    share the same tilt and azimuth values.
+
+    If only the GCR is known, feed GCR into ``trackers_vertical_length`` and
+    specify ``trackers_row_pitch=1``.
+
+    .. versionadded:: 0.10.5
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar position zenith, in degrees.
+    solar_azimuth : numeric
+        Solar position azimuth, in degrees.
+    trackers_axis_azimuth : numeric
+        In degrees. North=0º, South=180º, East=90º, West=270º.
+    shaded_tracker_tilt : numeric
+        Tilt of the tracker receiving the shade in degrees.
```

Comment:
pvlib's convention is to use `tilt` as a positive (or zero) number, but the angles these equations accept can be positive or negative.  Can we call them `rotation` instead of `tilt`?

Reply:
I think we should also mention somewhere that both rotation parameters are right-handed rotations according to the specified `axis_azimuth` and `axis_tilt`.